src/ast/mod.rs

```diff
@@ -1080,6 +1090,12 @@ pub enum Action {
     /// Be wary! Only delete entries that are subsumed in some way or
     /// guaranteed to be not useful.
     Delete(Symbol, Vec<Expr>),
+    /// Set an entry to be `unextractable`
```

Comment:
I think this documentation should clarify what an "entry" is.
What we mean is an "e-node" or a constructor and it's children e-classes.
In particular, any e-node that becomes equal to it will also be unextractable